src/cost.py

```diff
@@ -5,13 +5,21 @@
 from src.read import read_csv
 
 
-def cost(source):
+class KS:
     """
-     cost method
-     """
-    total_cost = 0
-    data = read_csv(source)
-    dataframe = pd.DataFrame(data)
-    for item in dataframe.columns:
-        total_cost += dataframe[item]
-    return total_cost
+       Knapsack class
+    """
+
+    def __str__(self):
+        return self.__class__.__name__
+
+    def cost(self):
```

Comment:
self is supposed to be the instance, like _this_ in java.
Please use a `__init__(self, params)` to create a constructor for the KSProblem class!
The class should contain should store the related data in a field -> cost() should not have to read from csv